tests/test-components/build.rs

```diff
@@ -25,9 +25,14 @@ fn main() {
     for package in packages {
         let crate_path = PathBuf::from("components").join(&package);
         let manifest_path = crate_path.join("Cargo.toml");
+        if !manifest_path.exists() {
```

Comment:
Nit: can we log when the path doesn't exist? The logs don't show unless a verbosity option is passed, but it could be nice to provide some sort of indication that a directory is being skipped.